### PR TITLE
vsv: Security advisory for VSV13

### PR DESCRIPTION
**Pull request by Dridi**
This is a draft optimistically targeting next Monday for the releases.

It should be merged after the three releases are published on the homepage repository.

We can probably produce a circuit breaker mitigation in VCL with inline C, I will try that early on Monday.

### CODE REVIEW

File: R1/source/security/VSV00013.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+* Varnish Enterprise by Varnish Software version 6.0.12r1.
+
+Mitigation
+----------
+
+If upgrading Varnish is not possible, it is possible to mitigate the problem
+by simply disabling HTTP/2 support::
+
+    varnishadm param.set feature -http2
```
Comment (nigoroll) [inline R1/source/security/VSV00013.rst:50]: Point to ALPN?

**Review by nigoroll — COMMENTED** [review]
Excellent writeup except for one point

File: R1/source/security/VSV00013.rst (hunk 1 above)
Comment (Dridi) [reply R1/source/security/VSV00013.rst:50]: Doesn't https://github.com/varnishcache/varnish-cache/issues/2389 make that a moot point? We always need an upgrade from HTTP/1 anyway, don't we?

Comment (nigoroll) [reply R1/source/security/VSV00013.rst:50]: What happens when the h2 ALPN is present, but not supported?

File: R1/source/security/VSV00013.rst [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+None of the members of the Varnish projects were part of the one month
+embargo of the CVE, so we learned about this vulnerability when it was
+disclosed.
+
+Varnish Software immediately assessed the vulnerability and started two
+efforts in parallel. Dag Haavi Finstad implemented the Rapid Reset Attack
+detection and Dridi Boukelmoune implemented the VCL circuit breaker.
+
+Nils Goroll implemented *vmod_h2*, and reviewed the two mitigations along
```
Comment (nigoroll) [inline R1/source/security/VSV00013.rst:129]: As Varnish Software is mentioned, I would appreciate if I was mentioned as "Nils Goroll of UPLEX".

Comment (Dridi) [reply R1/source/security/VSV00013.rst:129]: Good point, my bad!

File: R1/source/security/VSV00013.rst (hunk 1 above)
Comment (simonvik) [reply R1/source/security/VSV00013.rst:50]: It fails in curl, chrome and firefox.

File: R1/source/security/VSV00013.rst (hunk 2 above)
Comment (Dridi) [reply R1/source/security/VSV00013.rst]: Done.

**Comment by Dridi** [comment]
Merged as part of #120.

**Pull request closed without merging**